scripts/tcl_commands/sta.tcl

```diff
@@ -48,4 +62,62 @@ proc run_sta {args} {
     exec echo "[TIMER::get_runtime]" | python3 $::env(SCRIPTS_DIR)/write_runtime.py "sta - openroad"
 }
 
+proc run_parasitics_sta {args} {
+    set options {
+        {-sdf_out optional}
+    }
+    set flags {}
+    parse_key_args "parasitics_sta" args arg_values $options flags_map $flags
+
+    set_if_unset arg_values(-sdf_out) [file rootname $::env(CURRENT_DEF)].sdf
+    set_if_unset arg_values(-spef_out_prefix) [file rootname $::env(CURRENT_DEF)]
```

Comment:
`spef_out_prefix` isn't declared in the `options` variable. I am unsure if passing this argument, when calling the function, will have any effect.